src/create-template-orders.js

```diff
@@ -266,6 +252,9 @@ function _generateTemplateOrderImportDraft(
     nextReminderDateISOString = nextDeliveryCronDate.toISOString()
   }
   const templateOrder = {
```

Comment:
I figured out some fields are missing here, those are: `customLineItems`, `taxedPrice`, `orderState`, `shipmentState`, `paymentState`, `shippingInfo` ? Do you think all of those are not necessary ? and if yes, why ? 🤔 

and another one, should we set: `completedAt` ?

Reply:
This is a creation of a template order from checkout order. In the checkout order, you can also have non-subscription type of (custom) line items. Let's say you have wine subscription + pants in the checkout order.
If you think about what they represent in the checkout order, it doesn't make much sense to copy the values. WDYT?

`completedAt` - not sure if we need this value. It is only for order import and when you create checkout order, you don't have this value.

Reply:
I see, but not sure, the question is related to the additional data that might be missing when required on the creation of the subscription-order (in case, the original checkout-order cleaned up after some time because of the lots of orders) and I think in concept we said we will copy all values, that wonders me that we are missing some fields here. Can we open this to a discussion in slack ?

Reply:
I don't see the reason why should we copy these fields. For example `taxedPrice` is a price for all line items, including the ones which are not subscriptions. Let's say for pants + wine subscription is 300 USD, but for wine subscription it is only 150 USD. If I copy over this field, it would show 300 USD, which doesn't make sense. The other fields are similar - they are related to the checkout order, but doesn't make sense for template orders anymore.